staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -1670,24 +1676,30 @@ func (az *Cloud) reconcileLoadBalancerRule(
 				},
 			})
 		} else if port.Protocol != v1.ProtocolUDP && port.Protocol != v1.ProtocolSCTP {
-			// we only add the expected probe if we're doing TCP
+			var actualPath *string
+			var probePort int32
+
+			probePort = port.NodePort
+
 			if probeProtocol == "" {
-				probeProtocol = string(*probeProto)
+				probeProtocol = string(network.ProbeProtocolHTTP)
```

Comment:
Might want a comment here? Where did `probProto` come from before and why is this being changed to HTTP

Reply:
Done